src/cea708.c

```diff
@@ -64,6 +66,10 @@ void cea708_parse_user_data_type_structure(const uint8_t* data, size_t size, use
     user_data->em_data = data[1];
     data += 2, size -= 2;
 
+    if (detail && size/3 != user_data->cc_count){
```

Comment:
Why a third? Did you mean to right shift by 3?

Reply:
cc_count is the number of sections of data (each section has the data itself and an extra metadata byte). This totals 3 bytes per section and size is the number of bytes in total.